server/tools/docker-entrypoint.sh

```diff
@@ -38,6 +39,9 @@ file_env 'KEYCLOAK_PASSWORD'
 
 if [[ -n ${KEYCLOAK_USER:-} && -n ${KEYCLOAK_PASSWORD:-} ]]; then
     /opt/jboss/keycloak/bin/add-user-keycloak.sh --user "$KEYCLOAK_USER" --password "$KEYCLOAK_PASSWORD"
+else
+    DEFAULT_PASSWORD=$(openssl rand -base64 20 | cut -c-20)
```

Comment:
Move this ```DEFAULT_PASSWORD``` variable initialization above (at line 31, after the initialization of the ```DEFAULT_USER``` variable).